package/network/services/dnsmasq/files/dhcp.conf

```diff
@@ -15,7 +15,7 @@ config dnsmasq
 	option leasefile	'/tmp/dhcp.leases'
 	option resolvfile	'/tmp/resolv.conf.d/resolv.conf.auto'
 	#list server		'/mycompany.local/1.2.3.4'
-	option nonwildcard	1 # bind to & keep track of interfaces
+	option bind		'dynamic' # bind to & keep track of interfaces
```

Comment:
Shouldn't a migration script be added to update existing configs similar to package/network/services/dnsmasq/files/50-dnsmasq-migrate-resolv-conf-auto.sh?